### PR TITLE
Fixing Patch and Delete For v1 API Group

### PR DESCRIPTION
**Pull request by seanmalloy**
Commit 216e7dd8bfc0b3f4459c7f178e9136ad97484f32 broke patching and
deleting resouces in the v1 API group(i.e. Service Pod, Namespace).

### CODE REVIEW

**Comment by seanmalloy** [comment]
Code has not been tested yet.

**Comment by seanmalloy** [comment]
Confirmed working.

**Review by vinny-sabatini — APPROVED** [review]

File: library/openshift_provision.py [hunk 1]
```diff
@@ -1467,15 +1467,25 @@ def provision(self):
         # Perform action on resource
         resource_fqdn = self.resource['kind'] + "." + self.resource['apiVersion'].split("/")[0]
```
Comment (sanbornick) [inline library/openshift_provision.py:1468]:
```suggestion
        if self.resource['apiVersion'].split("/")[0] == "v1":
                resource_fqdn = self.resource['kind']
            else:
                resource_fqdn = self.resource['kind'] + "." + self.resource['apiVersion'].split("/")[0]
```

Refactor the name if it is not pleasing. Then remove other changes?

**Review by sanbornick — COMMENTED** [review]
I'd go about it slightly differently. I'm still scratching my head a bit on why this fails, i still feel like we are missing something

**Pull request merged**